.github/workflows/build-and-test.yml

```diff
@@ -22,47 +22,29 @@ jobs:
     with:
       runsonlabels: '["macOS", "self-hosted", "spezi"]'
       scheme: SpeziBluetooth-Package
+      swiftVersion: 6
       artifactname: SpeziBluetooth-Package.xcresult
       resultBundle: SpeziBluetooth-Package.xcresult
-  packageios_latest:
-    name: Build and Test Swift Package iOS Latest
-    uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    with:
-      runsonlabels: '["macOS", "self-hosted", "spezi"]'
-      scheme: SpeziBluetooth-Package
-      xcodeversion: latest
-      swiftVersion: 6
-      artifactname: SpeziBluetooth-Package-Latest.xcresult
-      resultBundle: SpeziBluetooth-Package-Latest.xcresult
   ios:
     name: Build and Test iOS
     uses: StanfordSpezi/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
     with:
       runsonlabels: '["macOS", "self-hosted", "spezi"]'
       path: 'Tests/UITests'
       scheme: TestApp
+      swiftVersion: 6
```

Comment:
Remove in case we are moving to Swift 6 by default anyways. I am not 100% sure this even does a difference here?